common/men_at_arms_types/temp_draft_doc.txt

```diff
@@ -0,0 +1,604 @@
+﻿@cultural_maa_extra_ai_score = 60 # Equivalent to having 6 extra regiments beyond what the code scoring would indicate (see NEGATIVE_SCORE_PER_EXISTING_REGIMENT)
+#Centuar
+deathcaller = {
+	type = caster
+	
+	damage = 45
+	toughness = 40
+	pursuit = 20
+	screen = 20
+	
+	terrain_bonus = {
+		desert = { damage = 20 toughness = 20 pursuit = 12 screen = 12 }
+		desert_mountains = { damage = 12 toughness = 12 pursuit = 8 screen = 8}
+		drylands = { damage = 20 toughness = 20 pursuit = 15 screen = 10}
+	}
+
+	counters = {
+		light_aerial = 1
+		massive_infantry = 1
+		heavy_infantry = 1
+	}
+
+	buy_cost = { gold = cost_120_recruitment_cost }
+	low_maintenance_cost = { gold = cost_120_low_maint_cost }
+	high_maintenance_cost = { gold = cost_120_high_maint_cost }
+	
+	stack = 75
+	ai_quality = { value = @cultural_maa_extra_ai_score }
+	icon = deathcaller
+}
+
+impaler = {
+	type = archers
+	
+	damage = 30
+	toughness = 25
+	pursuit = 20
+	screen = 15
+	
+	terrain_bonus = {
+		plains = { damage = 12 toughness = 12 }
+		drylands = { damage = 12 toughness = 12 }
+	}
+
+	counters = {
+		skirmishers = 1
+		light_aerial = 1
+		casters = 1
+	}
+
+	buy_cost = { gold = cost_105_recruitment_cost }
+	low_maintenance_cost = { gold = cost_105_low_maint_cost }
+	high_maintenance_cost = { gold = cost_105_high_maint_cost }
+	
+	stack = 100
+	ai_quality = { value = @cultural_maa_extra_ai_score }
+	icon = impaler
+}
+
+#Draenei
+##TBD
+#shatari = {
+#	type = caster
+	
+#	damage = 75
+#	toughness = 20
+#	pursuit = 15
+#	screen = 10
+	
+#	terrain_bonus = {
+#		hills = { toughness = 5 }
+#		plains = { damage = 5 pursuit = 5 }
+#		drylands = { damage = 5 pursuit = 5}
+#	}
+
+#	counters = {
+#		pikemen = 2
+#		massive_infantry = 2
+#		heavy_infantry = 2
+#	}
+
+#	buy_cost = { gold = cost_120_recruitment_cost }
+#	low_maintenance_cost = { gold = cost_120_low_maint_cost }
+#	high_maintenance_cost = { gold = cost_120_high_maint_cost }
+	
+#	stack = 75
+#	ai_quality = { value = @cultural_maa_extra_ai_score }
+#	icon = shatari
+#}
+
```

Comment:
Will this be done before merging the PR? If not, I'd delete this for now.

Reply:
no, it wont at all most likely. Once every stat is good I will move everything over and delete whatever is not needed

Reply:
This will be kept on spreadsheet if necessary in the future, I will delete once stats approved. I will resolve this, nothing to change atm.